.github/workflows/ci.yml

```diff
@@ -172,38 +160,29 @@ jobs:
       client_tag: ${{ matrix.client.tag }}
 
   perf-tests:
-    # Only the debug images have perf tooling
-    if: ${{ github.event_name == 'pull_request' }}
-    name: perf-tests-${{ matrix.version.prefix }}-${{ matrix.test_name }}
-    needs:
-      - build-base-perf-artifacts
-      - build-head-perf-artifacts
+    name: perf-tests
+    needs: build-perf-artifacts
     runs-on: ubuntu-22.04
     permissions:
       contents: read
       id-token: write
       pull-requests: write
     env:
       API_IMAGE: 'us-east1-docker.pkg.dev/firezone-staging/firezone/api'
-      API_TAG: ${{ matrix.version.sha }}
+      API_TAG: ${{ github.sha }}
       WEB_IMAGE: 'us-east1-docker.pkg.dev/firezone-staging/firezone/web'
-      WEB_TAG: ${{ matrix.version.sha }}
+      WEB_TAG: ${{ github.sha }}
       ELIXIR_IMAGE: 'us-east1-docker.pkg.dev/firezone-staging/firezone/elixir'
-      ELIXIR_TAG: ${{ matrix.version.sha }}
+      ELIXIR_TAG: ${{ github.sha }}
       GATEWAY_IMAGE: 'us-east1-docker.pkg.dev/firezone-staging/firezone/perf/gateway'
-      GATEWAY_TAG: ${{ matrix.version.sha }}
+      GATEWAY_TAG: ${{ github.sha }}
       CLIENT_IMAGE: 'us-east1-docker.pkg.dev/firezone-staging/firezone/perf/client'
-      CLIENT_TAG: ${{ matrix.version.sha }}
+      CLIENT_TAG: ${{ github.sha }}
       RELAY_IMAGE: 'us-east1-docker.pkg.dev/firezone-staging/firezone/perf/relay'
-      RELAY_TAG: ${{ matrix.version.sha }}
+      RELAY_TAG: ${{ github.sha }}
     strategy:
       fail-fast: false
       matrix:
-        version:
-          - sha: ${{ github.sha }}
-            prefix: head
-          - sha: ${{ github.event.pull_request.base.sha }}
-            prefix: base
         test_name:
```

Comment:
Can we not remove `scripts/perf` entirely and all these tests here? Or are they required for bencher to run?

Reply:
We have to define the different iperf runs somewhere. I think the shell-scripts plus a matrix is more maintainable?